config-vm.md

```diff
@@ -0,0 +1,37 @@
+# Virtual-machine-specific Configuration
+
+Virtual-machine-based runtimes require additional configuration to that specified in the [default spec configuration](config.md).
+
+This optional configuration is specified in a "VM" object:
+
+* **`imagePath`** (string, required) path to file that represents the root filesystem for the virtual machine.
+* **`kernel`** (object, required) specifies details of the kernel to boot the virtual machine with.
+
+Note that `imagePath` refers to a path on the host (outside of the virtual machine).
+This field is distinct from the **`path`** field in the (#Root-Configuration) section since in the context of a virtual-machine-based runtime:
+
+* **`ImagePath`** will represent the root filesystem for the virtual machine.
+* The container root filesystem specified by **`path`** will be mounted inside the virtual machine at a location chosen by the virtual-machine-based runtime.
```

Comment:
This line isn't clear to me. Looking at example below, it implies that `vmlinuz` carries the rootfs of the container which doesn't sound right to me.